go/oasis-test-runner/env/env.go

```diff
@@ -17,6 +19,21 @@ var ErrEarlyTerm = errors.New("env: sub-process exited early")
 // CleanupFn is the cleanup hook function prototype.
 type CleanupFn func()
 
+// TestInstanceInfo contains information of the current test run.
+type TestInstanceInfo struct {
+	// Test is the name of the test.
+	Test string `json:"test"`
+
+	// Instance is the instance name of the test. e.g. oasis-test-runner123456
```

Comment:
Does this need to be globally unique? If so, it should probably also include a hostname or something.

Reply:
Yeah, this should be globally unique, however it seems really small chance of 6-digit random number collision in few weeks (Prometheus history), if we are to run daily benchmarks few times a day. I rather left it descriptive i.e. the name of the folder. Is hostname always unique each build? If so, we could just prepend it.